doc/record.md

```diff
@@ -0,0 +1,100 @@
+# Record format
+Records are intended to make exchange of RDF safer and easier. The design has arised when working with facility design data, but we expect it to be useful for many cases. Specifically, use cases with high-velocity append-only data is probably not a good match. 
```

Comment:
>high velocity append-only data is probably not a good match
Does this refer to what might be otherwise called "time series data"?
I propose exemplifying a "good match" as well as the "bad match" already mentioned.

Good matches (in my estimation):
- Management of change for heterogenous datasets of arbitrary interconnectedness
- Storage and interchange of data within or between large organizations
- Multi-disciplinary analysis of data across domain verticals

Bad matches:
- Performance sensitive handling of _large_ datasets of small, well-known, fixed schema of low interconnectedness
    - Matrix data
    - Sensor streams
- Ephemeral data (i.e. data which is no longer needed after consumption)

Reply:
Ok har lagt det inn i motivation.md